lib/readyset.rb

```diff
@@ -23,7 +24,7 @@ def self.current_config
     configuration.inspect
   end
 
-  # Executes a raw SQL query against ReadySet. The query is sanitized prior to being executed.
+  # Executes a raw SQL query against Readyset. The query is sanitized prior to being executed.
```

Comment:
nit: I think we should continue to stylize "ReadySet" as camelcase in comments and documentation, since that's how it's stylized on the website